pystac/serialization/identify.py

```diff
@@ -28,9 +28,9 @@ def __init__(self, object_type, version_range, common_extensions, custom_extensi
         self.custom_extensions = custom_extensions
 
     def __repr__(self):
-        return '<{} {} common_ext={} custom_ext{}>'.format(self.object_type, self.version_range,
-                                                           ','.format(self.common_extensions),
-                                                           ','.format(self.custom_extensions))
+        return '<{} {} common_ext={} custom_ext={}>'.format(self.object_type, self.version_range,
```

Comment:
This is a bug - should be a `join` instead of a `format`: 

```python
def __repr__(self):
        return '<{} {} common_ext={} custom_ext={}>'.format(
            self.object_type, 
            self.version_range,
            ','.join(self.common_extensions),
            ','.join(self.custom_extensions))

```

Reply:
Fixed in 0d1d5e18e8e7661c9a7d3c4c0593de2d02ddb3cf.

Reply:
Thanks!